server/prisma/schema.prisma

```diff
@@ -63,19 +63,35 @@ model events {
   rsvps            rsvps[]
   tags             tags[]
   user_event_roles user_event_roles[]
+  event_reminders  event_reminders[]
 }
 
 model rsvps {
+  created_at       DateTime            @default(now())
+  updated_at       DateTime            @updatedAt
+  user_id          Int
+  event_id         Int
+  date             DateTime
+  on_waitlist      Boolean
+  confirmed_at     DateTime?
+  canceled         Boolean
+  events           events              @relation(fields: [event_id], references: [id], onDelete: Cascade, onUpdate: NoAction)
+  user             users               @relation(fields: [user_id], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  event_reminder   event_reminders[]
+
+  @@id([user_id, event_id])
+}
+
+model event_reminders {
   created_at   DateTime  @default(now())
   updated_at   DateTime  @updatedAt
   user_id      Int
   event_id     Int
-  date         DateTime
-  on_waitlist  Boolean
-  confirmed_at DateTime?
-  canceled     Boolean
-  events       events    @relation(fields: [event_id], references: [id], onDelete: Cascade, onUpdate: NoAction)
-  user         users     @relation(fields: [user_id], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  remind_at    DateTime
+  notified     Boolean   @default(false)
```

Comment:
```suggestion
  notifying     Boolean   @default(false)
```
Bit of a nitpick, but I think this is clearer.  Reason being, the reminder can be in basically three states: "initial", "notifying" and "done".  Initially `notifying` would be `false`, then when a reminder is being processed `notifying` is set to `true` and finally the reminder is done once its record is deleted.